about.html

```diff
@@ -0,0 +1,161 @@
+                        consectetur adipisicing elit. Necessitatibus, dolorem dolor. Beatae exercitationem in earum
+                        consectetur
+                        odit mollitia dignissimos quaerat commodi consequuntur, voluptatem repellendus quas expedita,
+                        architecto
+                        optio recusandae dolor!</p>
+                    <p class="about-main-content">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+            </div>    
+            </div>
+        </div>
+    </section>
+    <!--end main section-->
+
+    <!--about info section-->
+    <section id="about-info">
+        <div id="about-info-container">
+            <h2 id="about-info-second-title">Web Summit Lisbon 2023</h2>
+            <hr>
+                        <p class="about-main-content"> Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate placeat
+                            illum
+                            saepe, accusantium eveniet at adipisci iste nesciunt sapiente. Lorem ipsum dolor sit amet consectetur
+                            adipisicing elit. Necessitatibus, dolorem dolor. Beatae exercitationem in earum consectetur odit
+                            mollitia
+                            dignissimos quaerat commodi consequuntur, voluptatem repellendus quas expedita, architecto optio
+                            recusandae
+                            dolor!</p>
+                            <p class="about-main-content">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora est modi nemo aperiam ullam distinctio quaerat esse pariatur sit! Vero, iure nisi. Veritatis et quas, quaerat magni ad ab voluptatem! Lorem ipsum dolor sit amet consectetur adipisicing elit. Eveniet eaque id dolorum fugit minus nostrum sunt esse ullam, quas quis doloremque ab in quam omnis asperiores blanditiis nobis necessitatibus quasi.</p>
+                        <p class="about-main-content">Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio voluptatem
+                            doloremque aut corporis molestias, voluptates voluptatibus itaque perferendis libero sapiente quis
+                            aspernatur quaerat dolor eaque autem nesciunt perspiciatis inventore. Cupiditate.</p>
+                        <p class="about-main-content">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+                    <img src="/images-capstone/web-summit-logo.png" alt="about me section logo"/>
+        </div>
+        <p class="about-main-content">Lorem ipsum dolor sit amet consectetur adipisicing elit:</p>
+        <p><a href="mailto:websummitlisbon@gmail.com" class="about-email">websummitlisbon@gmail.com</a></p>
+    </section>
+    <!--end of about info section-->
+
+        <!--past events section-->
+<section id="past-events">
+    <div id="past-events-container">
+        <h2 class="see-past-events">See past CC Global Summits events</h2>
```

Comment:
- [ ] Looks like one text is over another text. Please fix that.

![Screenshot from 2022-06-24 02-59-40](https://user-images.githubusercontent.com/26767383/175406343-8739523a-d3fd-45d3-8ec8-313ab64cf808.png)

Reply:
thank you so much for the feedback @Deepakdanger ! Can I ask you in which resolution are you testing the project? This is what I see in the default mobile resolution (375px -718px). Please let me know 
![Screenshot from 2022-06-23 18-57-43](https://user-images.githubusercontent.com/37988027/175428765-11d7ad2c-9fcf-4e74-a166-3357a9cbbc4c.png)